apps/system/js/remote/message_controller.js

```diff
@@ -40,7 +46,8 @@
 
     _handle_message: function(evt) {
       var data = evt.data;
-      if (data.target !== this.displayId) {
+      if (parseInt(data.target) !== this.displayId &&
+          parseInt(data.target) !== -1) {
```

Comment:
Do you mean a message contains `data.target === -1` will be accepted by all "remote systems"?